packages/tpetra/core/test/CrsMatrix/CrsMatrix_UnitTests2.cpp

```diff
@@ -430,6 +431,28 @@ inline void tupleToArray(Array<T> &arr, const tuple &tup)
     } else {
       TEST_COMPARE_FLOATING_ARRAYS(norms,zeros,MT::zero());
     }
+
+    // test the constructors based on 4 maps + local matri_crsx
+    RCP<MAT> tri_crs_2 = rcp(new MAT(tri_crs->getRowMap(), tri_crs->getColMap(), tri_crs->getDomainMap(),
+            tri_crs->getRangeMap(), tri_crs->getLocalMatrix()) );
+    TEST_EQUALITY(tri_crs_2->isFillComplete(), true);
+    auto exporter = tri_crs_2->getGraph()->getExporter();
+    auto importer = tri_crs_2->getGraph()->getImporter();
+    TEST_EQUALITY(tri_crs->getDomainMap()->isSameAs(*(importer->getSourceMap())), true);
+    TEST_EQUALITY(tri_crs->getColMap()   ->isSameAs(*(importer->getTargetMap())), true);
+    TEST_EQUALITY(tri_crs->getRowMap()   ->isSameAs(*(exporter->getSourceMap())), true);
```

Comment:
Is the Export object trivial (and therefore null) here?  You may want to protect these equality tests with `if (! importer.is_null ())` resp. `if (! exporter.is_null ())`.